### ISSUE TITLE
Cannot reload workers in zerg mode

### ISSUE THREAD

**Issue by soulmerge** [unbit/uwsgi#854]
I am using the following configuration to load a zergpool server and a zerg worker:

    [overlord]
    master
    socket = overlord.socket
    zerg-server = zerg.socket
    protocol = http

    [zergling]
    master
    zerg = zerg.socket
    master-fifo = zergling.fifo

Loading the two processes works fine:

    uwsgi --ini uwsgi.ini:overlord
    uwsgi --ini uwsgi.ini:zergling

However, when I try to reload the zergling (by writing `r` into the master fifo or sending a SIGHUP), the process dies leaving the following error message at the end:

    The -s/--socket option is missing and stdin is not a socket.

**Comment by unbit** [comment]
Yes, gracefully reloading when in zerg mode is basically broken. The main problem is that there is no way to match an inherited file descriptor as a zerg (as the name of the socket is not the one of the zerg server but the one of the true socket). This is generally not a problem as zerg instances are expected to continuosly die and reborn, but having graceful reload would be great too.

I leave the issue open, hoping for a technical solution.

Thanks
